docs/datalayer.md

```diff
@@ -56,3 +56,53 @@ $happn._mesh.datalayer.events.on('detatch', function(ev) {
 });
 
 ```
+
+### Datalayer Compaction
+
+The datalayer can be set to compact at an interval, or can also be compacted by a call:
+
+*the following code demonstrates an interval compaction configuration:*
+```javascript
+var config = {
+  datalayer: {
+    port: 55007,
+    filename:test_file_interval,
+    compactInterval:10000//compact every 10 seconds
```

Comment:
millisecond resolution on this is awkward at the application level, where we would typically want to set intervals of minutes, hours or days. On the other hand, milliseconds values are very conventional. Unsure...